include/plugin_interface/SC_InlineBinaryOp.h

```diff
@@ -381,16 +382,16 @@ inline int sc_lcm(int a, int b)
 
 inline long sc_gcd(long a, long b)
 {
-	long t;
+	
+	if (a == 0) return b;
+	if (b == 0) return a;
 
+	long t;
```

Comment:
oh jesus, i finally noticed where this variable is used. the better solution would be to declare it in-scope, in lines 403 and 405 below.